CI is broken due to kustomize missing Namespace creation

There's an odd behavior during the CI: `kustomize` doesn't create the expected Namespace (`capsule-system`).

```
cd config/manager && /snap/bin/kustomize edit set image controller=quay.io/clastix/capsule:latest
/snap/bin/kustomize build config/default | kubectl apply -f -
namespace/system created
customresourcedefinition.apiextensions.k8s.io/tenants.capsule.clastix.io created
clusterrole.rbac.authorization.k8s.io/capsule-proxy-role created
clusterrole.rbac.authorization.k8s.io/capsule-metrics-reader created
clusterrolebinding.rbac.authorization.k8s.io/capsule-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/capsule-proxy-rolebinding created
mutatingwebhookconfiguration.admissionregistration.k8s.io/capsule-mutating-webhook-configuration created
validatingwebhookconfiguration.admissionregistration.k8s.io/capsule-validating-webhook-configuration created
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
Error from server (NotFound): error when creating "STDIN": namespaces "capsule-system" not found
Makefile:51: recipe for target 'deploy' failed
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -44,8 +44,9 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-build-
           ${{ runner.os }}-
-    # Installing Ginkgo
-    - name: installing Ginkgo
+    - name: Removing kustomize
+      run: sudo snap remove kustomize
+    - name: Installing Ginkgo
       run: go get github.com/onsi/ginkgo/ginkgo
     - uses: actions/setup-go@v2
       with:

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -31,13 +31,15 @@ jobs:
     runs-on: ubuntu-18.04
     steps:
     - uses: actions/checkout@master
-    # Cache Go modules
-    - name: Cache Go Modules
+    - name: Cache Go modules and Docker images
       uses: actions/cache@v1
       env:
-        cache-name: cache-go-modules
+        cache-name: gomod-docker
       with:
-        path: ~/go/pkg/mod
+        path: |
+          ~/go/pkg/mod
+          /var/lib/docker
+          /home/runner/work/capsule/capsule
         key: ${{ runner.os }}-build-${{ env.cache-name }}
         restore-keys: |
           ${{ runner.os }}-build-

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -28,7 +28,7 @@ jobs:
         args: --timeout 2m
   kind:
     name: e2e
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
     steps:
     - uses: actions/checkout@master
     # Cache Go modules

